wifiphisher/pywifiphisher.py

```diff
@@ -136,6 +142,14 @@ def check_args(args):
             W +
             '] Pre-shared key must be between 8 and 63 printable characters.')
 
+    if args.handshake_capture and not os.path.isfile(
+            args.handshake_capture):
+        sys.exit('[' +
+                 R +
+                 '-' +
+                 W +
+                 '] handshake capture does not exist.')
+
```

Comment:
Maybe we can add an additional check here. One that checks that the give pcap file does have a handshake capture that we can use to verify the captured PSK.